cypress/integration/1228-landing_page/1242-request_draft_doi_button_is_disabled_for_publication_with_existing_doi/disabled_request_doi_button.js

```diff
@@ -0,0 +1,39 @@
+// Feature: Request/Draft DOI button is disabled for Publications with existing DOI
+
+import { USER_DRAFT_DOI } from '../../../support/constants';
+import { my_registrations_tabs, landing_page_buttons } from '../../../support/data_testid_constants';
+
+const registration_titles = {

```

Comment:
Sneket seg inn litt python-casing her..?

```suggestion
import { myRegistrationsTabs, landingPageButtons } from '../../../support/data_testid_constants';

const registrationTitles = {

```

Reply:
ser det gjelder mange av filene i PR her, vet ikke om det er bevisst eller ikke

Reply:
Har jobbet en del med python i det siste så det er sikker ubevisst